config/invenio.conf

```diff
@@ -1942,6 +1942,12 @@ CFG_BATCHUPLOADER_WEB_ROBOT_RIGHTS = {
 # By default all DOIs are excluded
 CFG_BIBUPLOAD_INTERNAL_DOI_PATTERN = [^\w\W]
 
+# CFG_BIBUPLOAD_DOI_CHECKING - check multi DOIs
+# Parameter for switch on / off DOI checking in bibuploads
+# CFG_BIBUPLOAD_DOI_CHECKING = 0: multi DOIs are possible
+# CFG_BIBUPLOAD_DOI_CHECKING = 1: multi DOIs are forbidden (default)
+CFG_BIBUPLOAD_DOI_CHECKING = 1
```

Comment:
What about `True/False` instead `1/0`?
Like in https://github.com/inveniosoftware/invenio/blob/legacy/config/invenio.conf#L673